src/main/java/com/commercetools/sync/products/templates/beforeupdatecallback/SyncSingleLocale.java

```diff
@@ -26,7 +27,7 @@
 import static java.util.stream.Collectors.toList;
 import static java.util.stream.Collectors.toMap;
 
-final class SyncSingleLocale {
+public final class SyncSingleLocale {
```

Comment:
do we need a release note update for like exposed?

Reply:
The thing is it's not meant to be used publicly. It's just there as a template (that's why in the `/templates/` package) that can be copied.